packages/react-app/src/components/sidebar.tsx

```diff
@@ -30,13 +29,12 @@ export default function Sidebar() {
           <Outset gu8>
             {page === ENUM_PAGES.List && (
               <>
+                <QuestModal questMode={ENUM_QUEST_VIEW_MODE.Create} />
```

Comment:
We need to be connected to see that so we have to check for account && QuestModal